lib/pos_epoch_complete_hook.go

```diff
@@ -138,21 +138,20 @@ func (bav *UtxoView) runEpochCompleteSnapshotGeneration(epochNumber uint64) erro
 	return nil
 }
 
-// Updates the currentEpochEntry to the next epoch's.
-func (bav *UtxoView) runEpochCompleteEpochRollover(epochNumber uint64, blockHeight uint64, view uint64, blockTimestampNanoSecs uint64) error {
+func (bav *UtxoView) computeNextEpochEntry(epochNumber uint64, blockHeight uint64, view uint64, blockTimestampNanoSecs uint64) (*EpochEntry, error) {
```

Comment:
Definitely good to clarify that the passed in block height is the provided epoch's initial block height. Otherwise, it's ambiguious
```suggestion
func (bav *UtxoView) computeNextEpochEntry(epochNumber uint64, initialBlockHeight uint64, view uint64, blockTimestampNanoSecs uint64) (*EpochEntry, error) {
```